tests/foreman/ui/test_repository.py

```diff
@@ -714,6 +714,35 @@ def test_positive_sync_ansible_collection_gallaxy_repo(session, module_prod):
         assert result['result'] == 'success'
 
 
+@pytest.mark.tier2
+def test_positive_no_errors_on_repo_scan(target_sat):
+    """Scan repos for RHEL Server Extras, then check the production log
+    for a specific error
+
+    :id: 443bf4af-7f9a-48b8-8f98-fdb170e8ae88
+
+    :expectedresults: The specific error isn't contained in the prod log
+
+    :customerscenario: True
+
+    :BZ: 1994212
+
+    :CaseLevel: Integration
+    """
+    org = target_sat.api.Organization().create()
+    manifests.upload_manifest_locked(org.id)
```

Comment:
qq, here, I would like to know why you selected upload_manifest_locked?

Reply:
This should be refactored using new manifester fixtures / helpers !

The explicit locking is not required now as we have fixture scoped locking, so just fixture is enough ! 

CC: @synkd